### PR TITLE
Add ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by chasinglogic**
We have had a few issues [like this one](https://github.com/mongodb/homebrew-brew/issues/27) that are not actually related to the code in this repository and should go through our community resources. 

This template will add a disclaimer for users to read when opening their issue. Similar to the SCons project's issue template.

### CODE REVIEW

**Comment by acmorrow** [comment]
LGTM. Lets see if this helps. If not, we can just disable issues here and add to the README.

**Comment by chasinglogic** [comment]
I've changed this to include the support link in the README and I'm going to disable Github issues after merge.

@acmorrow PTAL

File: README.md [hunk 1]
```diff
@@ -85,8 +85,10 @@ To shutdown `mongod` started manually, use the `admin` database and run `db.shut
...
 ```
 
-## Additional Information
+## Additional Information and Problem Reporting
 
 This tap was created using the Homebrew documentation on [How to Create and Maintain a tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md).
 
 You can find additional information in the [Homebrew project README](https://github.com/Homebrew/brew#homebrew).
+
+If you're having issues with MongoDB please check out [our community support resources](https://www.mongodb.com/community-support-resources)
```
Comment (acmorrow) [inline README.md]: If we are going to disable issues here (and I think I support that, having looked at the issues already filed), I think we should add some language here that points people to the correct place to file issues with the homebrew tap (the SERVER project, presumably). That community support resources page doesn't link to JIRA anywhere.

Comment (chasinglogic) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -85,8 +85,10 @@ To shutdown `mongod` started manually, use the `admin` database and run `db.shut
...
 ```
 
-## Additional Information
+## Additional Information and Problem Reporting
 
 This tap was created using the Homebrew documentation on [How to Create and Maintain a tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md).
 
 You can find additional information in the [Homebrew project README](https://github.com/Homebrew/brew#homebrew).
+
+If you're having issues with MongoDB please check out [our community support resources](https://www.mongodb.com/community-support-resources) if you've found a bug please [open a JIRA ticket in the SERVER project](https://jira.mongodb.org)
```
Comment (acmorrow) [inline README.md]: Make it two sentences and end the second with a period. Then LGTM.

Comment (chasinglogic) [reply README.md]: Done.

**Pull request merged**